doc/div-alg.xml

```diff
@@ -584,15 +583,20 @@ containing the character <C>Irr(G)[n]</C>.
 <![CDATA[
 gap> G:=SmallGroup(72,21);
 <pc group of size 72 with 5 generators>
-gap> D:=DefectGroupOfConjugacyClassAtP(G,18,3);
+gap> for i in [1..Length(Irr(G))] do 
+>      if Size(KernelOfCharacter(Irr(G)[i])) then 
```

Comment:
This example fails the test. Should it be `if Size(KernelOfCharacter(Irr(G)[i]))=1`  ?